src/mobile/src/ui/views/wallet/exchanges/MoonPay/AddAmount.js

```diff
@@ -416,7 +432,7 @@ class AddAmount extends Component {
         } = this.props;
```

Comment:
To do: Amount should be validated with something like `Number(this.parseAmount(amount)) !== NaN`